merchant-sdk/example-app/README.md

```diff
@@ -1,11 +1,11 @@
 Gini Merchant SDK Example App
-=========================
+=============================
 
 This example app provides you with a sample usage of the Gini Merchant SDK.
 
-The Gini Health API Library is used for analyzing documents and creating payment requests.
+The Gini Merchant API Library is used for analyzing documents and creating payment requests.
```

Comment:
Hi @a-szotyori , should we change `Health API` to `Merchant API` at this point? Or only after we duplicate? 

Not sure clients need to know how it's working under the hood, so your approach could be better. Just asking because I've intentionally left it as `Health API` everywhere, and would need to go back and change it.

Reply:
Thank you for pointing this out! So as far as I know go-live will be with using `https://merchant-api.gini.net`, but through the Health API Library by changing the base url to `merchant-api.gini.net`. 

Without refactoring we can't hide the Health API Library from client devs because it needs to be instantiated for [authentication](https://developer.gini.net/gini-mobile-android/health-sdk/sdk/4.1.0/html/authentication.html) and passed to `GiniMerchant`.

I will change the name of Merchant API Library back to Health API Library. I did the renaming too quickly without thinking about its effects.